src/ott/neural/models/models.py

```diff
@@ -403,3 +409,261 @@ def tree_flatten(self) -> Tuple[Sequence[Any], Dict[str, Any]]:  # noqa: D102
         "rng": self.rng,
         "state": self.state
     }
+
+
+class Block(nn.Module):
+  dim: int = 128
+  num_layers: int = 3
+  act_fn: Any = nn.silu
+  out_dim: int = 32
+
+  @nn.compact
+  def __call__(self, x):
+    for _ in range(self.num_layers):
+      x = nn.Dense(self.dim)(x)
+      x = self.act_fn(x)
+    return nn.Dense(self.out_dim)(x)
+
+
+class NeuralVectorField(BaseNeuralVectorField):
```

Comment:
same, would drop `Neural`, 

maybe call `VelocityField` if tied more clearly to an ODE